guide/src/continuous-integration.md

```diff
@@ -0,0 +1,38 @@
+# Continuous integration
+
+This chapter shows some examples how to use Yarner with Continuous integration.
+There are always two ways to get Yarner:
+
+1. Use a rust image and install it using `cargo`
```

Comment:
I would not recommend this at all. For a user of Yarner, it should not matter at all in which language it is implemented and if his/her project does not use Rust, he should not include this into its CI due to the diligence required to keep a CI rolling at all.

Reply:
Removed it.